django/db/models/functions/base.py

```diff
@@ -201,10 +201,7 @@ def __init__(self, expression, substring, **extra):
         expression: the name of a field, or an expression returning a string
         substring: a string to find inside expression
```

Comment:
Good point, done. Also made the docs use `import Value as V` which is what all the other db function docs do. This made me think that the `V` alias could be provided by `django.db.models` immediately to avoid the `as V` pattern because some style guides discourage import aliases, so I posted to the developers mailing list around providing this.